src/components/LayoutHeader/Header.js

```diff
@@ -165,6 +165,25 @@ const Header = ({location}: {location: Location}) => (
             to="/versions">
             v{version}
           </Link>
+          <Link
+            css={{
+              padding: '5px 10px',
+              whiteSpace: 'nowrap',
+              ...fonts.small,
+
+              ':hover': {
+                color: colors.brand,
+              },
+
+              ':focus': {
+                outline: 0,
+                backgroundColor: colors.lighter,
+                borderRadius: 15,
+              },
+            }}
+            to="/languages">
+            {language}
```

Comment:
Maybe it's better to call this "Translations" or "Languages" like Vue does?

Reply:
I don't understand what you're suggesting. Are you referring to the route? It's already "/languages". None of our routes use capitalization.

If you're referring to the page, its header is already "Languages".

Reply:
Oh, I understand what you mean. Show "Languages" rather than showing the _current_ language.